src/main/java/jenkins/plugins/office365connector/Webhook.java

```diff
@@ -183,6 +196,12 @@ public static class DescriptorImpl extends Descriptor<Webhook> {
         private String globalUrl;
         private String globalName;
 
+        private String ip;
+        private String username;
```

Comment:
do we need to keep same values for Proxy and here?

Reply:
yes, this is needed to capture the proxy info